sbn/codingconv/CodingConventions.md

```diff
@@ -133,10 +133,10 @@ even at the cost of some additional key strokes, and then to minimize
 possible name collisions or ambiguities.
 
 **[R]** Use of a descriptive control variable is **required**
-  in any loop longer than five lines.
+  in any loop longer than five lines, unless implementing a mathematical formula (matrix operations?) where i,j,k indices are traditional
```

Comment:
Somehow in a matrix operation I do consider `i`, `j` and `k` descriptive, or `mu`, `nu`, lambda`, `rho` for tensors, and `x`, `y` and `z` when there is only one vector around. Do you thing we need to spell an exception out?

Reply:
I'm worried the original suggestion will have people calling their variables `firstMatrixSubscript` and so on, so I think it's probably better to be explicit (though could combine with the part where you talk about physics equations)